_format/1.2/index.md

```diff
@@ -2091,13 +2091,19 @@ family.
 
 A "query parameter family" is the set of all query parameters whose name starts
 with a "base name", followed by zero or more instances of empty square brackets
-(i.e. `[]`) or square-bracketed legal [member names]. The family is referred to
+(i.e. `[]`), square-bracketed legal [member names] or square-bracketed
```

Comment:
Based on skimming the start of the spec, it's using Oxford comma, so there should be a comma before `or`.